common/source/docs/common-buttons.rst

```diff
@@ -4,6 +4,9 @@
 Buttons
 =======
 
+.. warning::
```

Comment:
buttons are configured as inputs,not outputs so dont need this warning

Reply:
Fixed along with adding same for buzzer. Agree on the other various functions - I took a look at a solid sample of those and came to the same conclusion.